javaclient/src/main/java/confluo/PropertiesParser.java

```diff
@@ -0,0 +1,32 @@
+package confluo;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+public class PropertiesParser {
+
+    /**
+     *  parse unique domain properties
```

Comment:
We use sentence case for documentation, i.e., capitalize first letter of each sentence. Please use this consistently across all documentation.

Reply:
done